### PR TITLE
Add commas subsection to PHP section

### PR DESCRIPTION
**Pull request by dealancer**

### CODE REVIEW

File: codeguide.md [hunk 1]
```diff
@@ -110,6 +110,25 @@ echo $a['b'] . ',' . $c . ',' . $d . "\n\n"; // Correct
...
 
+### Commas
+
+- Do not use comma with the last item of an array.
+```
+$my_array = array(
+[tab]'foo'   => 'somevalue',
+[tab]'foo2'  => 'somevalue2',
+[tab]'foo3'  => 'somevalue3',
+[tab]'foo34' => 'somevalue3', // Correct
```
Comment (thatdevgirl) [inline codeguide.md]: I think this is backwards. This is the incorrect example.

File: codeguide.md [hunk 2]
```diff
@@ -110,6 +110,25 @@ echo $a['b'] . ',' . $c . ',' . $d . "\n\n"; // Correct
...
+);
+
+$my_array = array(
+[tab]'foo'   => 'somevalue',
+[tab]'foo2'  => 'somevalue2',
+[tab]'foo3'  => 'somevalue3',
+[tab]'foo34' => 'somevalue3' // Incorrect
```
Comment (thatdevgirl) [inline codeguide.md]: This is the correct example.

**Comment by dealancer** [comment]
@thatdevgirl whoops, fixed.

**Comment by thatdevgirl** [comment]
@dealancer Perfect - thank you!

**Pull request merged**